boards/efm32g-st700/include/board.h

```diff
@@ -0,0 +1,65 @@
+
+
```

Comment:
@ryankurte yes and the doxygen part. There is an example in the [wiki](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#documentation)